exercises/saddle-points/description.md

```diff
@@ -21,5 +21,7 @@ A matrix may have zero or more saddle points.
 Your code should be able to provide the (possibly empty) list of all the
 saddle points for any given matrix.
 
+The matrix can have different a number of rows and columns (Non square).
```

Comment:
Should be:
The matrix can have *a* different number of rows and columns (non-square).